specification/paths/Users.json

```diff
@@ -133,14 +133,14 @@
               "meta": {
                 "type": "object",
                 "required": [
-                  "client_id"
+                  "broker_id"
                 ],
                 "properties": {
                   "client_id": {
```

Comment:
```suggestion
                  "broker_id": {
```

Reply:
Oops :sweat_smile: This worked because additional properties are allowed.